static/js/biomedical/disease/page.tsx

```diff
@@ -211,19 +235,27 @@ export class Page extends React.Component<PagePropType, PageStateType> {
     );
   }
   private fetchData(): void {
-    axios.get("/api/disease/" + this.props.dcid).then((resp) => {
-      this.setState({
-        diseaseGeneAssociation: getDiseaseGeneAssociation(resp.data),
-        diseaseSymptomAssociation: getDiseaseSymptomAssociation(resp.data),
-        chemicalCompoundDiseaseTreatment: getCompoundDiseaseTreatment(
-          resp.data
-        ),
-        chemicalCompoundDiseaseContraindication:
-          getCompoundDiseaseContraindication(resp.data),
-        diseaseCommonName: getDiseaseCommonName(resp.data),
-        diseasePrevalenceIDexists: doesDiseasePrevalenceIDexist(resp.data),
-        dataFetched: true,
-      });
-    });
+    const diseasePromise = axios
+      .get("/api/disease/" + this.props.dcid)
+      .then((resp) => resp.data);
+    const diseaseParentPromise = axios
+      .get("/api/disease/diseaseParent/" + this.props.dcid)
```

Comment:
nit: sorry, didn't notice this last time, but when naming endpoints, we want to use hyphens, so instead of `diseaseParent`, we want `disease-parent`

Reply:
No worries! Fixed it!